docs/website/blog/2022-09-01-odo-v3-rc1.md

```diff
@@ -0,0 +1,111 @@
+- Fix: ServiceBinding resources are not deployed with odo deploy [\#6029](https://github.com/redhat-developer/odo/pull/6029) ([valaparthvi](https://github.com/valaparthvi))
+- odo dev: Cleanup after displaying the error [\#6024](https://github.com/redhat-developer/odo/pull/6024) ([feloy](https://github.com/feloy))
+- Fix: Non odo components not reported [\#6021](https://github.com/redhat-developer/odo/pull/6021) ([valaparthvi](https://github.com/valaparthvi))
+- Watch for devfile dependencies [\#6020](https://github.com/redhat-developer/odo/pull/6020) ([feloy](https://github.com/feloy))
+- Restart port forwarding on failure [\#6013](https://github.com/redhat-developer/odo/pull/6013) ([feloy](https://github.com/feloy))
+- Remove command: odo project [\#6008](https://github.com/redhat-developer/odo/pull/6008) ([valaparthvi](https://github.com/valaparthvi))
+- Enhance error message when user runs from a non-component directory [\#6006](https://github.com/redhat-developer/odo/pull/6006) ([valaparthvi](https://github.com/valaparthvi))
+- Allow specifying message to end the spinner with [\#5972](https://github.com/redhat-developer/odo/pull/5972) ([dharmit](https://github.com/dharmit))
+
+**Documentation:**
+
+- Update site index page with lighter font [\#6035](https://github.com/redhat-developer/odo/pull/6035) ([cdrage](https://github.com/cdrage))
+-  Add blog post for v3.0.0-beta3 and update installation docs [\#6032](https://github.com/redhat-developer/odo/pull/6032) ([valaparthvi](https://github.com/valaparthvi))
+- Add blog post for v3.0.0-beta3 and update installation docs [\#6019](https://github.com/redhat-developer/odo/pull/6019) ([valaparthvi](https://github.com/valaparthvi))
+- Document how odo works with projects/namespaces [\#6009](https://github.com/redhat-developer/odo/pull/6009) ([valaparthvi](https://github.com/valaparthvi))
+
+**Testing/CI:**
+
+- fix flake and use operatorhub to install EDB [\#6004](https://github.com/redhat-developer/odo/pull/6004) ([anandrkskd](https://github.com/anandrkskd))
+
+**Merged pull requests:**
+
+- Bump to version 3.0.0-rc1 [\#6064](https://github.com/redhat-developer/odo/pull/6064) ([feloy](https://github.com/feloy))
+- Update script to support alpha/beta/rc versions [\#6063](https://github.com/redhat-developer/odo/pull/6063) ([feloy](https://github.com/feloy))
+- update to OpenShift 4.11 [\#6060](https://github.com/redhat-developer/odo/pull/6060) ([kadel](https://github.com/kadel))
+- Troubleshoot IBM Cloud Windows tests failing [\#6038](https://github.com/redhat-developer/odo/pull/6038) ([feloy](https://github.com/feloy))
+- test: use `T.Setenv` to set env vars in tests [\#6037](https://github.com/redhat-developer/odo/pull/6037) ([Juneezee](https://github.com/Juneezee))
+- Revert "Add blog post for v3.0.0-beta3 and update installation docs" [\#6030](https://github.com/redhat-developer/odo/pull/6030) ([cdrage](https://github.com/cdrage))
+
+## Note about Release Candidate
```

Comment:
Maybe this section could be the first one, before listing the notable changes? It currently looks a bit "hidden" to me..